### PR TITLE
Added route profile on BurgerMenu

### PR DESCRIPTION
**Pull request by LauraCourquin**

### CODE REVIEW

**Review by ghost — APPROVED** [review]
ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/components/Burger/index.jsx [hunk 1]
```diff
@@ -3,11 +3,14 @@ import contact from "@assets/pictureNavMob/contact.png";
...
 import search from "@assets/pictureNavMob/Search.svg";
 import { Link, useNavigate } from "react-router-dom";
-import { useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
 import { toast } from "react-toastify";
 import SBurger from "./style";
 
 function Burger() {
+  const { type } = useSelector((state) => state.user);
+  const { id } = useSelector((state) => state.user);
```
Comment (LoicBrassart) [inline frontend/src/components/Burger/index.jsx:12]:
```
  const { id,type } = useSelector((state) => state.user);
```

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**